compiler/src/test/java/com/squareup/anvil/compiler/codegen/ContributesSubcomponentHandlerGeneratorTest.kt

```diff
@@ -1773,6 +1773,78 @@ class ContributesSubcomponentHandlerGeneratorTest {
     }
   }
 
+  @Test fun `a contributed subcomponent can be replaced by a generated subcomponent`() {
```

Comment:
This name sounds more like the use-case is supported to me. What about `a previously generated contributed subcomponent cannot be replaced in a later round of generations`? Or at least `can` -> `can't`

Reply:
I forgot to update the name. I did TDD and initially planned to implement the functionality, but it was quite tricky and I stopped. Since this is such an edge case I added an explicit check to ensure it isn't supported and then forgot to update the method name.